components/keystone/aio-values.yaml

```diff
@@ -21,6 +21,12 @@ bootstrap:
           --user="${OS_USERNAME}" \
           --domain="${OS_DEFAULT_DOMAIN}" \
           "admin"
+    # create a demo user with demo password
+    openstack user create --domain="${OS_DEFAULT_DOMAIN}" --password demo demo
+    # create undercloud-dev group
+    openstack group create --or-show undercloud-dev
+    # add demo user to undercloud-dev group
+    openstack group add user undercloud-dev demo
```

Comment:
Can the `group add user` be run multiple times? Cause the bootstrap job does get restarted periodically.

Reply:
Yes, I tested it out first:
```
root@openstack-admin-client:/# openstack group add user undercloud-dev keekz
root@openstack-admin-client:/# openstack group add user undercloud-dev keekz
root@openstack-admin-client:/# echo $?
0
```
Some subcommands have `--or-show` flag, but this particular one doesn't.

Reply:
```bash
openstack group contains user undercloud-dev demo 2>&1| grep -q 'not in group' && openstack group add user undercloud-dev demo
```

should do the trick but guess the end result is the same so we can just leave it as-is